docs/superpowers/plans/2026-04-26-schedules-pg-net.md

```
@@ -0,0 +1,528 @@
+
+- [ ] **Step 3.2: Run the test and verify it fails**
+
+Run: `cd backend && npx vitest run tests/unit/schedule.service.logs-filter.test.ts`
+Expected: FAIL — both regex assertions fail because the current code does not include `AND settled = TRUE`.
+
+- [ ] **Step 3.3: Make the minimal source change**
+
+Open `backend/src/services/schedules/schedule.service.ts` and edit the `getExecutionLogs` method (around lines 379–411):
+
+- Change the row query `WHERE` clause from:
+  ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)